music21/duration.py

```diff
@@ -454,15 +457,17 @@ def quarterLengthToTuplet(qLen,
             qLenBase = opFrac(typeValue / float(i))
             # try multiples of the tuplet division, from 1 to max-1
             for m in range(1, i):
-                qLenCandidate = qLenBase * m
-                if qLenCandidate == qLen:
-                    tupletDuration = durationTupleFromTypeDots(typeKey, 0)
-                    newTuplet = Tuplet(numberNotesActual=i,
-                                       numberNotesNormal=m,
-                                       durationActual=tupletDuration,
-                                       durationNormal=tupletDuration,)
-                    post.append(newTuplet)
-                    break
+                for number_of_dots in POSSIBLE_DOTS_IN_TUPLETS:
+                    dot_multiplayer = DOT_LENGTH_MULTIPLAYER ** number_of_dots
```

Comment:
spelling, but this is not the correct formula for dot length multiplier -- for instance 2 dots would give 9/4 instead of correct 7/4 -- use the existing function in `duration` to get the multiplier for an arbitrary number of dots.  Feel free to check up to 2 or 3 dots.

Reply:
I used the `common.dotMultiplier` function for dots calculation. Added support double 0,1,2 dots. Didn't had any unit test example for a triple dotted tuplet note in mind, so I kept the 3 dots out of the picture for now.

Reply:
3 dots is too much for people to expect that we'll get it right, so that's fine.